code/modules/economy/account.dm

```diff
@@ -13,6 +14,7 @@
 	account_holder = newname
 	account_job = job
 	account_id = rand(111111,999999)
+	account_password = rand(0000,9999)
```

Comment:
This can generate a password like 37 or 5. The 4 zeroes do nothing. Should I make it so it requires the leading zeroes?

Reply:
I actually made this a text var instead of a num var, lets see how this works out